streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/ApplicationServiceImpl.java

```diff
@@ -519,6 +529,26 @@ public boolean existsByTeamId(Long teamId) {
         return baseMapper.existsByTeamId(teamId);
     }
 
+    @Override
+    public boolean existsRunningJobByClusterId(Long clusterId) {
+        boolean exists = baseMapper.existsRunningJobByClusterId(clusterId);
+        if (!exists) {
+            for (Application application : FlinkTrackingTask.getAllTrackingApp().values()) {
+                if (clusterId.equals(application.getFlinkClusterId())) {
+                    if (FlinkAppState.RUNNING.equals(application.getFlinkAppStateEnum())) {
+                        return true;
+                    }
+                }
+            }
+        }
+        return exists;
+    }
```

Comment:
This is bad case about code style. You can take a look Flink Code style in your free time, it's useful for  high quality code of StreamPark.

https://flink.apache.org/contributing/code-style-and-quality-common.html

<img width="1032" alt="image" src="https://user-images.githubusercontent.com/38427477/204298610-923c0359-d9d4-4f49-9718-8a15da59bf1a.png">

<img width="733" alt="image" src="https://user-images.githubusercontent.com/38427477/204299004-b44d768f-cfba-4c96-b861-19d11fd97d5a.png">

Reply:
> This is bad case about code style. You can take a look Flink Code style in your free time, it's useful for high quality code of StreamPark.
> 
> https://flink.apache.org/contributing/code-style-and-quality-common.html
> 
> <img alt="image" width="1032" src="https://user-images.githubusercontent.com/38427477/204298610-923c0359-d9d4-4f49-9718-8a15da59bf1a.png">
> 
> <img alt="image" width="733" src="https://user-images.githubusercontent.com/38427477/204299004-b44d768f-cfba-4c96-b861-19d11fd97d5a.png">

hi fanrui:

Thanks for your feedback. If you read the code carefully, you will find that the situation is different from what you describe

the following code is improved:

```
@Override
    public boolean existsRunningJobByClusterId(Long clusterId) {
        boolean exists = baseMapper.existsRunningJobByClusterId(clusterId);
        if (!exists) {
            for (Application application : FlinkTrackingTask.getAllTrackingApp().values()) {

                if (clusterId.equals(application.getFlinkClusterId()) && FlinkAppState.RUNNING.equals(application.getFlinkAppStateEnum()) ) {
                        return true;
                }
            }
        }
        return exists;
    }
```

Reply:
How about this?

```
    @Override
    public boolean existsRunningJobByClusterId(Long clusterId) {
        boolean exists = baseMapper.existsRunningJobByClusterId(clusterId);
        if (exists) {
            return true;
        }

        for (Application application : FlinkTrackingTask.getAllTrackingApp().values()) {
            if (clusterId.equals(application.getFlinkClusterId()) && FlinkAppState.RUNNING.equals(application.getFlinkAppStateEnum()) ) {
                return true;
            }
        }
        return false;
    }
```